skabase/SKALogger/SKALogger.py

```diff
@@ -272,8 +277,11 @@ def SetStorageLoggingLevel(self, argin):
                 dev_proxy = DeviceProxy(storage_logging_device[i])
                 dev_proxy.storageLoggingLevel = storage_logging_level[i]
                 i += 1
```

Comment:
Ditto

Reply:
Sorry.. Corrected. Commit #705831b58ecfa5cbf8a2fb7610047c649c603de8 addresses this.